checkstyle-configuration.xml

```diff
@@ -334,9 +334,4 @@
         <property name="format" value="\n\n\n" />
         <property name="message" value="Extra newline." />
     </module>
-
-    <module name="RegexpMultiline">
```

Comment:
@motlin @nikhilnanivadekar Are you both ok with this removal?

Reply:
Is this for windows development? I'm confused because even when I used to develop on windows, this rule didn't fire for me. I don't feel like digging into why, so I'm ok with removing it.

Reply:
@motlin I think it's because a) I'm on Windows and b) my Git is configured to check out files with platform-specific line endings. However, c) my Git is _also_ configured to check in Linux `\n` line endings, hence my commits pass on the server/didn't change anything. I feel like explicitly blacklisting `\r` makes development more difficult for new contributors and/or Windows users when they have the experience that I had where the build failed after cloning and doing nothing

Reply:
@motlin @jdimeo Someone else was complaining on collections-dev mail group they were having issues with checkstyle on Windows. Maybe related. I'm ok with this change.

Reply:
I tried to fix this checkstyle configuration in my local. After numerous attempts I am at a dead-end to have something work predictable. I am fine removing this change. Thanks @jdimeo 
Tagging @motlin and @donraab for transparency.